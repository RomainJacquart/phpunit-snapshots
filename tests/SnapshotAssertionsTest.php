<?php

namespace Madewithlove\PhpunitSnapshots;

class SnapshotAssertionsTest extends TestCase
{
    public function testCanUseSnapshotTesting()
    {
        $this->assertEqualsSnapshot(['foo', 'bar']);
        $this->assertEqualsSnapshot(['foo', 'baz']);
    }

    public function testCanEvenUseItInMultipleTests()
    {
        $this->assertEqualsSnapshot(['foo', 'bar'], 'some identifier');
    }
}
