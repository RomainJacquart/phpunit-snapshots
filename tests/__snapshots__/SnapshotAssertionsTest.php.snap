{
    "testCanUseSnapshotTesting-0": [
        "foo",
        "bar"
    ],
    "testCanUseSnapshotTesting-1": [
        "foo",
        "baz"
    ],
    "testCanEvenUseItInMultipleTests-0: some identifier": [
        "foo",
        "bar"
    ]
}